### PR TITLE
docs: using dayjs instead of moment

### PR DESCRIPTION
**Pull request by xiaohuoni**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] FAQ

### 🔗 Related issue link

https://github.com/ant-design/ant-design/issues/20446

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/calendar/index.en-US.md](https://github.com/ant-design/ant-design/blob/using-dayjs-instead-of-moment/components/calendar/index.en-US.md)
[View rendered components/calendar/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/using-dayjs-instead-of-moment/components/calendar/index.zh-CN.md)
[View rendered components/date-picker/index.en-US.md](https://github.com/ant-design/ant-design/blob/using-dayjs-instead-of-moment/components/date-picker/index.en-US.md)
[View rendered components/date-picker/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/using-dayjs-instead-of-moment/components/date-picker/index.zh-CN.md)
[View rendered components/time-picker/index.en-US.md](https://github.com/ant-design/ant-design/blob/using-dayjs-instead-of-moment/components/time-picker/index.en-US.md)
[View rendered components/time-picker/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/using-dayjs-instead-of-moment/components/time-picker/index.zh-CN.md)

### CODE REVIEW

**Comment by xiaohuoni** [comment]
@zombieJ 看看，这 lint 错误与本次提交无关
image

File: components/time-picker/index.zh-CN.md [hunk 1]
```diff
@@ -61,3 +61,83 @@ import moment from 'moment';
...
+import { TimePicker } from '@/components';
+import format from 'dayjs';
+```
+
+替换
+
+```js
+import { TimePicker } from 'antd';
+import format from 'moment';
+```
```
Comment (afc163) [inline components/time-picker/index.zh-CN.md]: 文档好复杂，后面有问题很难维护，能提成一个独立的包么。

Comment (xiaohuoni) [reply components/time-picker/index.zh-CN.md]: 可以的，但我觉得更好的是 antd 要不要提供一种模式切换？

Comment (xiaohuoni) [reply components/time-picker/index.zh-CN.md]: 或者 antd 直接使用 dayjs ？

Comment (zombieJ) [reply components/time-picker/index.zh-CN.md]: 单独搞一个页面呢？抽出来一个包还是依赖于 antd 的 DatePicker 样式，它其实也抽不干净。

Comment (afc163) [reply components/time-picker/index.zh-CN.md]: 单独搞一个页面吧，现在太散很容易改漏。

Comment (xiaohuoni) [reply components/time-picker/index.zh-CN.md]: 明白

**Pull request closed without merging**